apps/ai-content-generator/src/hooks/config/useInitializeParameters.ts

```diff
@@ -14,7 +14,7 @@ import { ParameterAction, ParameterReducer } from '@components/config/parameterR
 const useInitializeParameters = (dispatch: Dispatch<ParameterReducer>) => {
   const sdk = useSDK<ConfigAppSDK>();
 
-  const dispatchParameters = async () => {
+  const dispatchParameters = useCallback(async () => {
```

Comment:
Same comment as the `useSaveConfigHandler.ts`, does this resolve any potential re-rendering issue?